### ISSUE TITLE
redirect to proper login page after group signup

### ISSUE THREAD

**Issue by aguestuser** [affinityworks/main#496]
# Acceptance Criteria

* Given i am a new user who, when I successfully submitted a signup page to join a group, then I should be forwarded to a login page that it is possible for me to complete (ie: one that does not require me to already have an account and a password)

[STUB]

# Context

Currently, after #476, upon successfully signing up, I see this:

![476-post-submit-login](https://user-images.githubusercontent.com/6032844/35996534-9e5072aa-0ccb-11e8-882e-0f4fda1c297f.png)

(which is confusing)

This might be a duplicate of cards @Matt-Holland is already drawing up. If so: apologies! We can close it!

**Comment by aguestuser** [comment]
closing as duplicative of #512
